crates/cli/Cargo.toml

```
@@ -22,11 +22,13 @@ anvil_zksync_common.workspace = true
 anvil_zksync_traces.workspace = true
 
 zksync_types.workspace = true
+zksync_multivm.workspace = true
 zksync_telemetry.workspace = true
 alloy = { workspace = true, default-features = false, features = [
     "signer-mnemonic",
     "reqwest-rustls-tls",
 ] }
+reqwest.workspace = true
```

AFAIU your version does not disable default features, please do this:
```suggestion
reqwest = { workspace = true, default-features = false, features = ["json", "rustls-tls"] }
```
And revert the change to root `Cargo.toml`